src/Coders.cpp

```diff
@@ -0,0 +1,42 @@
+#include "Coders.hpp"
+#include <iostream>
+#include <string>
+
+
+Coders::Coders(std::string userName_
+             , std::string discordNickname_
+             , std::string githubLink_
+             , std::string firecodeLink_
+             , Group SetGroup_)
+             : userName(userName_)
+             , discordNickname(discordNickname_)
+             , githubLink(githubLink_)
+             , firecodeLink(firecodeLink_)
+             , SetGroup(SetGroup_){}
```

Comment:
This formatting is misleading and it's not obvious there we have arguments list and where initialization list. Why SetGroup starts with an uppercase?